### DISCUSSION TITLE
How can I set `resources.ignoreResourceUpdatesEnabled` in `argocd-cm` with the operator?

### DISCUSSION THREAD

**Discussion by choffa** [argoproj-labs/argocd-operator#1186] [Q&A] [answered]
I am trying to follow the [documentation on Reconciliation optimalization](https://argo-cd.readthedocs.io/en/stable/operator-manual/reconcile/#system-level-configuration). I am able to set `resource.customizations.ignoreResourceUpdates.<group>_<kind>` correctly using `argocd.spec.resourceIgnoreDifferences.resourceIdentifiers`.

However, the documentation notes that for this to work `resource.ignoreResourceUpdatesEnabled` must also be set to `"true"`. I cannot find any way to set this using the CRD. Does anyone know?

**Comment by andrii-korotkov-verkada** [comment]
You add `resource.ignoreResourceUpdatesEnabled: "true"` to `argocd-cm` data section. Same for `resource.customizations.<suffix>`.

**Comment by choffa** [comment, marked as answer]
I found the answer. @andrii-korotkov-verkada answer does not work as the operator generates the `argocd-cm` for me. However, the ArgoCD CRD has a `spec.extraConfig` field that is directly put in the generated `argocd-cm`.

Se also the doc: https://argocd-operator.readthedocs.io/en/latest/usage/extra-config/
